specification/devcenter/DevCenter/Environments/routes.tsp

```diff
@@ -122,10 +157,127 @@ interface Environments {
     }
   >;
 
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Missing key"
+  @doc("Gets Outputs from the environment.")
+  @route("/projects/{projectName}/users/{userId}/environments/{environmentName}/outputs")
+  @get
+  @added(APIVersions.v2024_02_01)
+  getOutputs is Azure.Core.Foundations.Operation<
+    {
+      @doc("The DevCenter Project upon which to execute operations.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      projectName: string;
+
+      @doc("The AAD object id of the user. If value is 'me', the identity is taken from the authentication context.")
+      @maxLength(36)
+      @minLength(2)
+      @pattern("^[a-zA-Z0-9]{8}-([a-zA-Z0-9]{4}-){3}[a-zA-Z0-9]{12}$|^me$")
+      @path
+      userId: string;
+
+      @doc("The name of the environment.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      environmentName: string;
```

Comment:
Same question as above, should all of these constraints also exist on the params in the patchEnvironment operation?

Reply:
Added the constraints